library/psalm-baseline.xml

```diff
@@ -4349,6 +4349,27 @@
       <code>func_get_args()</code>
     </MixedArgument>
   </file>
+  <file src="Ivoz/Provider/Domain/Model/VoicemailRelUser/VoicemailRelUser.php">
+    <MethodSignatureMismatch occurrences="1">
```

Comment:
Can we fix this issue?

Reply:
The problem here is that psalm complains about auto-generated stuff. Return type for Method setUser is static both on the interface and the abstract class, so actual types differ:

`ERROR: MethodSignatureMismatch - Ivoz/Provider/Domain/Model/VoicemailRelUser/VoicemailRelUser.php:11:7 - Method Ivoz\Provider\Domain\Model\VoicemailRelUser\VoicemailRelUserAbstract::setVoicemail with return type 'Ivoz\Provider\Domain\Model\VoicemailRelUser\VoicemailRelUserAbstract' is different to return type 'Ivoz\Provider\Domain\Model\VoicemailRelUser\VoicemailRelUserInterface' of inherited method Ivoz\Provider\Domain\Model\VoicemailRelUser\VoicemailRelUserInterface::setVoicemail (see https://psalm.dev/042)
class VoicemailRelUser extends VoicemailRelUserAbstract implements VoicemailRelUserInterface
`